bgpd/bgp_evpn_mh.c

```diff
@@ -1698,7 +1707,9 @@ static void bgp_evpn_es_vteps_show_detail(struct vty *vty,
 		if (!strlen(vtep_flag_str))
 			strlcat(vtep_flag_str, "-", sizeof(vtep_flag_str));
 
-		vty_out(vty, "  %s flags: %s", inet_ntoa(es_vtep->vtep_ip),
+		vty_out(vty, "  %s flags: %s",
+			inet_ntop(AF_INET, &es_vtep->vtep_ip, ip_buf,
```

Comment:
I think we can use the format %pI4 here

Reply:
I didn't realize vty_out allowed for %pI4 (assumed it was only for zlogs). I will check and fixup.

Reply:
Done